pkg/scheduler/objects/allocation_test.go

```diff
@@ -129,6 +130,8 @@ func TestNewAllocFromNilSI(t *testing.T) {
 func TestNewAllocFromSI(t *testing.T) {
 	res, err := resources.NewResourceFromConf(map[string]string{"first": "1"})
 	assert.NilError(t, err, "Resource creation failed")
+	tags := make(map[string]string)
+	tags[siCommon.CreationTime] = "100"
```

Comment:
can we define a fixed time in the past for the test,  and use that to make it simple to test and check:
```
const past = 1640995200 // 2022-1-1 00:00:00
tags[siCommon.CreationTime] =  strconv.Itoa(past)
```
Would also be good to see the negative test case tested: no time set and get a -1 back.